dev/AppNotifications/AppNotifications.idl

```diff
@@ -111,6 +111,22 @@ namespace Microsoft.Windows.AppNotifications
 
         // Gets or sets whether a Notification's pop-up UI is displayed on the user's screen.
         Boolean SuppressDisplay;
+
+        // Gets or sets the Notification Device Data
+        AppNotificationDevicesData DevicesData;
+    }
+
+    // The Notification Device Data
+    [contract(AppNotificationsContract, 1)]
```

Comment:
This should be 2
[contract(AppNotificationsContract, 2)]

Reply:
Can you please explain why? this is new interface rt..

Reply:
Yes, new should be +1 the previous as mentioned in the guidelines

Reply:
changed